src/components/Postform.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+
+class Postsform extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: '',
+            body: ''
+        };
+        this.onChange = this.onChange.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onChange(event) {
+        this.setState({ [event.target.name]: event.target.value});
+    }
+
+    onSubmit(event) {
+        event.preventDefault();
+        const post = {
+            title: this.state.title,
+            body: this.state.body
+        };
+
+        fetch('https://jsonplaceholder.typicode.com/posts', {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json'
+            },
+            body: JSON.stringify(post)
+        })
+          .then(res => res.json())
+          .then(data => console.log(data));
```

Comment:
This is just to see whether we are getting newly added post to the state. Operations will be performed using redux.